utility/common.go

```diff
@@ -13,3 +13,23 @@ func CleanUrlPath(p string) string {
 	url.Path = path.Clean(url.Path)
 	return url.String()
 }
+
+// InsertInMap modifies `data` *in place*, inserting `values` at the given `key` or at the top level if the `key`is "".
```

Comment:
This description is outdated.

Could you also add a note that `key` has to be a top-level key